code/_onclick/click.dm

```diff
@@ -300,6 +302,29 @@
  */
 /mob/proc/ranged_secondary_attack(atom/target, modifiers)
 
+/**
+ * Right click
+ *
+ * Used for right-clicking interactions, in similar fashion of AltClick.
+ * Returns [atom/proc/RightClick] on the atom being right-clicked. This sends a signal.
```

Comment:
Again, what does this return value *mean*? I shouldn't have to check other documentation to understand this one. It should have the same description of "returns TRUE if the click chain should not continue", or similar.

Sorry if this all seems like nit-picking, but accurate and useful documentation is super important.